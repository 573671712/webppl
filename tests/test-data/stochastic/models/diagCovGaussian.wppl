var model = function() {
  var x = sample(DiagCovGaussian({
    mu: Vector([1, 2]),
    sigma: Vector([3, 4])
  }));
  return T.sumreduce(x);
};
