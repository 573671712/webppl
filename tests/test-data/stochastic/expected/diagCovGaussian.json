{
  "mean": 3,
  "std": 5
}
